integration/tests/cook/test_basic.py

```diff
@@ -845,3 +845,33 @@ def get_docker_info():
             self.assertEqual(instance['ports'][1], int(ports['9090/udp'][0]['HostPort']))
         finally:
             util.session.delete('%s/rawscheduler?job=%s' % (self.cook_url, job_uuid))
+
+
+    def test_unscheduled_jobs(self):
+        job_uuid, resp = util.submit_job(self.cook_url,
+                                         command='ls',
+                                         constraints=[['HOSTNAME', 'EQUALS', 'fakehost']])
+        self.assertEqual(resp.status_code, 201, resp.content)
+        try:
+            unscheduled_resp = util.session.get('%s/unscheduled_jobs?job=%s' % (self.cook_url,
+                                                                                job_uuid))
```

Comment:
Nitpick: can we keep this on the previous line?

Reply:
Done